src/commands/strikes_command.py

```diff
@@ -32,12 +32,12 @@ async def add_strike(
     @bot.tree.command()
     @discord.app_commands.check(is_user_moderator)
     @discord.app_commands.describe(user="User whose strikes you are checking")
-    async def strike_details(interaction: discord.Interaction, user: discord.Member):
+    async def view_strikes(interaction: discord.Interaction, user: discord.Member):
         """Add a strike to the user"""
         async with create_response_context(interaction) as response_message:
             async with create_logging_embed(interaction, user=user) as logging_embed:
-                strike_details = await strike_service.get_user_strikes(
+                view_strikes = await strike_service.get_user_strikes(
                     logging_embed, user
                 )
 
-                response_message.set_string(strike_details)
+                response_message.set_string(view_strikes)
```

Comment:
we shouldn't change these two vars, just changing the function name is sufficient

Reply:
done